macros/parserDifferenceQuotient.pl

```diff
@@ -15,28 +15,27 @@
 ################################################################################
 
 =head1 NAME
-
 parserDifferenceQuotient.pl - An answer checker for difference quotients.
-
 =head1 DESCRIPTION
-
```

Comment:
Why are these blank lines being removed?  (Here and below.)  I particularly don't like the loss of the paragraph breaks below (original lines 30 and 37).  I'd also like to see a blank between the new lines 32 and 33.

Would you be willing to add those (back) in?